x/ccv/provider/keeper/relay.go

```diff
@@ -150,55 +153,93 @@ func (k Keeper) sendValidatorUpdates(ctx sdk.Context) {
 			// construct validator set change packet data
 			packetData := ccv.NewValidatorSetChangePacketData(valUpdates, valUpdateID, k.ConsumeSlashAcks(ctx, chainID))
 
-			// check whether there is an established CCV channel to this consumer chain
-			if channelID, found := k.GetChainToChannel(ctx, chainID); found {
-				// send this validator set change packet data to the consumer chain
-				err := utils.SendIBCPacket(
-					ctx,
-					k.scopedKeeper,
-					k.channelKeeper,
-					channelID,          // source channel id
-					ccv.ProviderPortID, // source port id
-					packetData.GetBytes(),
-					k.GetParams(ctx).CcvTimeoutPeriod,
-				)
-				if err != nil {
-					panic(fmt.Errorf("packet could not be sent over IBC: %w", err))
-				}
+			if !found {
+				// CCV channel not established:
+				// store the packet data to be sent once the CCV channel is established
+				k.AppendPendingVSCs(ctx, chainID, packetData)
+				return true // go to next consumer chain
+			}
+
```

Comment:
I think an easier to understand pattern for SendValidatorUpdates would be something similar to

```
func GetPendingVSCPackets(..) {
	packets := []Packet{}
	// read from store
	for bytes in decoded store data { // if nothing is in the store then nothing is iterated
		packets = append(packets, Unmarshal(bytes))
	}
	return packets
}

func ConsumePendingVSCPackets(..) {
	packets := GetPendingVSCPackets()
	DeletePendingVscPackets()
	return packets
}

func SendValidatorUpdates(..) {
        ...
	packets := ConsumePendingVSCPackets()
	if len(valUpdates) != 0 || len(unbondingOps) != 0 {
		//...
		packets = append(packets, newPackets)
	}
	TrySendPackets(packets)
}

```

Dittoing my other comment I think much of the packet code could be simpler and have less cases, but this would require refactoring of existing code, perhaps outside of the scope of the PR.